src/components/add-transaction-dialog/add-transaction-dialog.html

```diff
@@ -53,7 +53,7 @@ <h3 class="mdc-typography--subheading2">Type of transaction: </h3>
               <ul class="mdc-list mdc-menu__items">
                 <% for(let i=0; i<accounts.length; i++) {%>
                   <li class="mdc-list-item" role="option" tabindex="0">
-                    <%= accounts[i] %>
+                    <%= accounts[i].name %>
```

Comment:
can you send not only name, but currency and type? Select provides "string" value. So it should be in format that we can parse easily

Reply:
cancel this question